### ISSUE TITLE
Sword Art Online -Hollow Fragment- [PCSE00465]

### ISSUE THREAD

**Issue by StevenMiller123** [Vita3K/compatibility#1693]
# App summary
- App name: Sword Art Online -Hollow Fragment-
- App serial: PCSE00465
- App version: 1.00

# Vita3K summary
- Version: v0.1.8
- Build number: 3225
- Commit hash: 2f46b479
- CPU backend: Dynarmic
- GPU backend: OpenGL

# Test environment summary
- Tested by: StevenMiller123
- OS: Windows 11 22H2
- CPU: Intel Core I5-9600k
- GPU: Nvidia Geforce RTX 3060ti
- RAM: 16 GB

# Issues
- Overall performance issues
- The game renders in a small area
- Performance decreases significantly when rendering specific text (Look at the framerate in the last image)
- Vertex explosions/Lighting glitches
- User interface layering issues (Look at the minimap in the images)
- Graphical effect issues (Weird yellow filter covering the top of the screen)

# Screenshots
![Screenshot 2023-06-22 213331](https://github.com/Vita3K/compatibility/assets/56742918/ed8c37bc-3595-4695-aa51-1e1a8c751086)
![Screenshot 2023-06-22 213516](https://github.com/Vita3K/compatibility/assets/56742918/bd692b78-2281-4b31-903b-5af161551c5a)
![Screenshot 2023-06-22 214252](https://github.com/Vita3K/compatibility/assets/56742918/679687e6-96cc-4689-b1cf-9ef403df3a8d)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/11843488/vita3k.log)

# Recommended labels
- Ingame +
- Graphics bug
- slow

**Comment by jegesim** [comment]
![Screenshot_20231004-020641](https://github.com/Vita3K/compatibility/assets/146876496/944d5662-e50d-4207-84b6-af7f0d51a10e)
![Screenshot_20231004-020614](https://github.com/Vita3K/compatibility/assets/146876496/7a3bbf3a-0649-4851-b831-2609b95e8dfa)
Playable now without noticeable glitches

**Comment by StevenMiller123** [comment]
There is a slowdown bug that only occurs on the PC version of Vita3K, I will retest on a computer as soon as possible.

**Comment by StevenMiller123** [comment]
This may not be entirely representative of the current status of the game, as I am using an Intel Iris Xe iGPU to test right now. For me, the graphics are completely broken on OpenGL, however, the graphics on Vulkan are perfect now (was previously a black screen). Regardless of the renderer, the framerate slows down significantly when viewing rendered text. This framerate issue only happens on PC Vita3K, not the Android version of the emulator. 
![Screenshot 2023-10-03 173308](https://github.com/Vita3K/compatibility/assets/56742918/97b17097-301d-40b7-bed9-5369289eb678)
![Screenshot 2023-10-03 172443](https://github.com/Vita3K/compatibility/assets/56742918/3712a1c7-a219-4812-b780-5ea018368cd1)

TL:DR
The graphics bug is gone, but the game is still unreasonably slow (on any PC).

**Comment by SeongGino** [comment]
Runs well for me on both a PC and D900 device (though SAO:HF seems to use up all the available memory on the D900 device which causes Vita3k to get killed). On the Android device, sometimes some text takes a bit longer to load than usual, and some characters of text might be missing in dialogue. But it seems to be all fine on the PC (or mayhaps any Android device with more than 4GB RAM/2GB available).

**Comment by RetroGameTalk** [comment]
Very slow on an i7-11700KF and a RTX-3070 too.
I'm not saying my PC is strong but at least you have a baseline.
